js/src/sdk/base.toolset.ts

```diff
@@ -12,8 +12,22 @@ import { convertReqParams, converReqParamForActionExecution } from "./utils";
 import { ActionRegistry, CreateActionOptions } from "./actionRegistry";
 import { getUserDataJson } from "./utils/config";
 import { z } from "zod";
+
+export type ActionData = {
```

Comment:
Consider adding JSDoc comments for the `ActionData` type to document its purpose and the meaning of each field. This will improve code maintainability and help other developers understand the data structure better.